tests/Swarrot/Broker/MessageProvider/PhpAmqpLibMessageProviderTest.php

```diff
@@ -25,6 +26,36 @@ public function test_get_with_messages_in_queue_return_message()
         $this->assertInstanceOf('Swarrot\Broker\Message', $message);
     }
 
+    public function test_get_with_amqp_array_header_return_array_header()
+    {
+        if (class_exists(AMQPArray::class)) {
```

Comment:
It's better to mark test as skipped when the class doesn't exists.

it's easier to read than a huge if and it won't display a notice for an empty test if the class is not found.